docusaurus/docs-internals/editor-controls/multivalue-controls/iconsPicker.md

```diff
@@ -0,0 +1,271 @@
+  type: "iconsPicker",
+  devices: "desktop",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    },
+  ]
+}
+```
+
+The display is limited to responsive modes, specifically `tablet` and `mobile`.
+
+```js
+{
+  id: "icns",
+  type: "iconsPicker",
+  devices: "responsive",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    },
+  ]
+}
+```
+
+#### Disabled examples
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "icns",
+  type: "iconsPicker",
+  disabled: true,
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    },
+  ]
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" },
+      ],
+    },
+    {
+      id: "icns",
+      type: "iconsPicker",
+      disabled: videoType === "custom",
+      choices: [
+        {
+          title: "Item 1",
+          value: "val1",
+          icon: "nc-star",
+        },
+        {
+          title: "Item 2",
+          value: "val2",
+          icon: "nc-line",
+        },
+      ],
+    },
+  ];
+};
+```
+
+#### Helper examples
+
+The helper object contains a content property with the value `"Helper"`, which will be displayed as additional guidance or information for the user.
+
+```js
+{
+  id: "icns",
+  type: "iconsPicker",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    },
+  ],
+  helper: {
+    content: "Helper"
+  }
+}
+```
+
+When the helper object contains a position property with the value `"top-start"`, indicating that the helper text will be displayed at the top start of icon.
+
+```js
+{
+  id: "icns",
+  type: "iconsPicker",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    },
+  ],
+  helper: {
+    content: "Helper",
+    position: "top-start"
+  }
+}
+```
+
+### Return value
+
+The return value of the `iconsPicker` control is an array os strings, which is value of selected icons
+
+```js
+Array<string>
+```
+Example of value:
+```js
+["val1", "val2"];
+```
```

Comment:
1. your option returns an strigified array, so:

```ts
{
  value: string;
}
```

Example of value:
```ts
{
  value: '["val1", "val2"]'
}
```

2. `iconsPicker` also return the `active` key